lab-davide/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Looks good, I'm deducting a point because certain fields aren't used, make sure to specify your name and/or email in the author field (for example, "David Fazio <youremail@domain.com>").  Additionally since your test is done with mocha you can specify the "scripts" "test" command to be "mocha" (but if you do make sure to add "mocha" as a dev dependency).